jenkins/ml-models.JenkinsFile

```diff
@@ -28,17 +28,25 @@ pipeline {
         )
         }
     environment {
-        ARTIFACT_PATH = "${BASE_DOWNLOAD_PATH}/${VERSION}/"
+        ARTIFACT_PATH = "${env.BASE_DOWNLOAD_PATH == "ml-models/model_listing" ? "${BASE_DOWNLOAD_PATH}" : "${BASE_DOWNLOAD_PATH}/${VERSION}/"}"
```

Comment:
Should we add `.json` suffix?

Reply:
All the files part of `ml-models/model_listing` would be published @gaiksaya @peterzhuamazon correct me if i'm wrong.
Thank you

Reply:
I think we should add the suffix if it is just 1 file to avoid downloading and uploading any other files as a security measure. WDYT?

Reply:
Sorry missed this, yes good point.

Reply:
Just one point I believe there can be multiple JSON files under `ml-models/model_listing` that should be pushed without version, in that case adding suffix for 1 file would be the right solution and entire path should be allowed `ml-models/model_listing` @dhrubo-os will we have multiple files under `ml-models/model_listing` in future?

Reply:
May be for now we can point to the 1 file we are talking about here so that we don't release any other files accidentally. I'm not 100% sure if we will add another json file here or not. 

So may be we can update this code later if that requirement appears later. What do you think?

Reply:
Thanks pointed to one file`ml-models/model_listing/pre_trained_models.json`.